packages/example-contract/package.json

```diff
@@ -5,13 +5,14 @@
   "description": "",
   "author": "Fuel Labs <contact@fuel.sh> (https://fuel.network/)",
   "scripts": {
-    "build": "./scripts/build.sh"
+    "prebuild": "forc build",
+    "build": "node ./node_modules/fuels/dist/bin.js typegen -i out/debug/example-contract-abi.json -o src/example-contract-types"
```

Comment:
Just wondering why you opted to introduce a script here? I don't think we should be implementing multiple ways of interacting with the typegen CLI in case SDK consumers use this as a reference.

Reply:
Good catch! https://github.com/FuelLabs/fuels-ts/pull/984/commits/14082fa44187f4f25fcc85643a238598bcd6d070

I forgot to migrate the nomenclatures I used in the old PR.

Please, check the section `Build Optimizations` here:
 - https://github.com/FuelLabs/fuels-ts/pull/827

Let me know if you have any remaining considerations.